### DISCUSSION TITLE
v2/zh/guide/markdown/echarts

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#417] [Announcements]
# v2/zh/guide/markdown/echarts

让你 VuePress 站点中的 Markdown 文件支持图表。 此插件使用 ECharts 提供相应功能。

https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/echarts.html

**Comment by jaydiy** [comment]
不能传入变量，创建动态图表
